src/python/pants/backend/python/lint/mypy/rules_integration_test.py

```diff
@@ -225,32 +223,3 @@ def add(x: int, y: int) -> str:
         assert len(result) == 1
         assert result[0].exit_code == 1
         assert f"{self.package}/math/add.py:5" in result[0].stdout
-
-    @pytest.mark.skip(
-        reason="Figure out how to get MyPy working with both namespace packages and source roots. See https://github.com/Eric-Arellano/mypy-debug."
-    )
```

Comment:
See https://pantsbuild.slack.com/archives/CASMF8SJ1/p1593473127209200. I can't figure out how to get this working, even without Pants. So, I don't want dead code hanging around.

I may take a third look tomorrow..